src/storage/mongodb/UserStorage.ts

```diff
@@ -602,7 +602,7 @@ export default class UserStorage {
     }
     // Email
     if (params.email) {
-      filters.email = params.email;
+      filters.email = { $regex: `^${params.email}$`, $options: 'i' };
```

Comment:
I don't like the regex with an email that could contain special chars like '.' or @.
Here I want the exact match.